pkg/agent/agent.go

```diff
@@ -1197,11 +1206,37 @@ func (i *Initializer) initVMLocalConfig(nodeName string) error {
 		return err
 	}
 
+	// TODO: Handle for multiple interfaces
+	var uplinkInterface *net.Interface
+	foundNetDevice := false
+	for _, addr := range en.Spec.Interfaces[0].IPs {
+		var ipFilter *utilip.DualStackIPs
+		var err error
+		epIP := net.ParseIP(addr)
+		if epIP.To4() != nil {
+			ipFilter = &utilip.DualStackIPs{IPv4: epIP}
+		} else {
+			ipFilter = &utilip.DualStackIPs{IPv6: epIP}
+		}
+		_, _, uplinkInterface, err = util.GetIPNetDeviceFromIP(ipFilter, sets.NewString())
+		if err != nil {
+			klog.InfoS("Failed to get net device for", "IP", addr)
```

Comment:
klog.InfoS("Unable to get net device by IP", "IP", addr)

Reply:
Done